src/systems/systems/src/air_conditioning/acs_controller.rs

```diff
@@ -533,27 +538,48 @@ impl<const ZONES: usize> SimulationElement for ZoneController<ZONES> {
 }
 
 pub struct PackFlowValveSignal {
-    target_open_amount: [Ratio; 2],
+    target_open_amount: Ratio,
 }
 
-impl PackFlowValveSignal {
-    pub fn new(target_open_amount: [Ratio; 2]) -> Self {
+impl PneumaticValveSignal for PackFlowValveSignal {
+    fn new(target_open_amount: Ratio) -> Self {
         Self { target_open_amount }
     }
 
-    pub fn target_open_amount(&self, pack_id: usize) -> Ratio {
-        self.target_open_amount[pack_id - 1]
+    fn target_open_amount(&self) -> Ratio {
+        self.target_open_amount
+    }
+}
+
+#[derive(Clone, Copy)]
+pub enum PackId {
+    Pack1 = 0,
+    Pack2 = 1,
+}
```

Comment:
**thought**: I'm not sure if the usage of this enum is idiomatic. I would consider introducing a "new type" as an alternative, as this saves you from some of the `as` casting hassle:

```rust
pub struct Pack(usize);

impl From<usize> for Pack {
    fn from(value: usize) -> Self {
        Pack(value)
    }
}
```

Note that `#[repr(usize)]` is also helpful when you want to represent an enum with a numeric type greater than the minimum necessary size. Any `#[repr(u*)]` or `#[repr(i*)]` is allowed.

Reply:
Thanks for the review as always @davidwalschots! I've changed this now and followed your suggestion. I do like the idea but the downside is that you could essentially create a pack with any `id` you wanted, so I've added an `if` statement to prevent pack numbers different than 1 or 2 from being created (the A380 also has 2 packs). 

I've also added a `From` implementation for usize, as the pack number is always 1 higher than the pointer.

Reply:
Looks good! Regarding the `From<Pack> for usize` implementation, the `number-1` might cause confusion, as it makes `From` specific to one use case (index-based access). An alternative could be adding a function `to_index(&self) -> usize`, to ensure it's 100% clear there's a conversion to index happening. This would result in e.g.:
```rust
self.pack_flow_controller[pack_id.to_index()]
```

Reply:
Good idea, I've changed it now as suggested.